### PR TITLE
Added country code functionality

### PR DESCRIPTION
**Pull request by SaniyaKumawat123**
Added country code functionality:-
1. user can add country code
2. user can get all country code

### CODE REVIEW

File: src/main/java/com/tripply/booking/model/ResponseModel.java
Comment (Amitsinghongraph) [inline src/main/java/com/tripply/booking/model/ResponseModel.java:1]: Dont use constructor please use lombok annotations

Comment (2000-saniya) [reply src/main/java/com/tripply/booking/model/ResponseModel.java:1]: Yes, removed constructor, and used builder

Comment (SaniyaKumawat123) [reply src/main/java/com/tripply/booking/model/ResponseModel.java:1]: Yes, removed constructor, and used builder

**Review by Amitsinghongraph — APPROVED** [review]
Approve commit

**Pull request merged**